google-cloud-spanner/src/test/java/com/google/cloud/spanner/RetryOnInvalidatedSessionTest.java

```diff
@@ -226,327 +233,327 @@ public void setUp() throws InterruptedException {
               .build()
               .getService();
       client = spanner.getDatabaseClient(DatabaseId.of("[PROJECT]", "[INSTANCE]", "[DATABASE]"));
-      invalidateSessionPool(client, spanner.getOptions().getSessionPoolOptions().getMinSessions());
+      invalidateSessionPool();
     }
   }
 
-  private static void invalidateSessionPool(DatabaseClient client, int minSessions)
-      throws InterruptedException {
-    // Wait for all sessions to have been created, and then delete them.
-    Stopwatch watch = Stopwatch.createStarted();
-    while (((DatabaseClientImpl) client).pool.totalSessions() < minSessions) {
```

Comment:
This wait loop is no longer needed, as the session pool has a built-in option to wait for `MinSessions`.